src/datachain/client/local.py

```diff
@@ -75,23 +57,20 @@ def path_to_uri(cls, path: str) -> str:
 
     @classmethod
     def split_url(cls, url: str) -> tuple[str, str]:
-        """
```

Comment:
Should we reword the doc instead of removing this?

Reply:
Just added it in parent class.